src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,46 @@
 'use strict';
 
+const aliveAnimalsList = [];
+const HEALTH = 100;
+const BITE_POWER = 50;
+
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = HEALTH;
+    aliveAnimalsList.push(this);
+  }
+
+  static get alive() {
+    return aliveAnimalsList.filter(animal => animal.health > 0);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name) {
+    super(name);
+
+    this.hidden = false;
+  }
+
+  hide() {
+    this.hidden = !this.hidden;
```

Comment:
```suggestion
    this.hidden = true;
```

Reply:
I guess, we should not show the animal if we call method `hide` twice. To show animal we would need to create another method or rename this one